meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
It's not that verbose (subjective isn't it?), i'll try to record something with asciinema